src/rest/APIRequest.js

```diff
@@ -6,7 +6,7 @@ const AbortController = require('abort-controller');
 const fetch = require('node-fetch');
 const { UserAgent } = require('../util/Constants');
 
-if (https.Agent) var agent = new https.Agent({ keepAlive: true });
+const agent = new https.Agent({ keepAlive: true });
```

Comment:
Is this always defined?
```suggestion
const agent = new https.Agent?.({ keepAlive: true });
```

Reply:
It should always be defined 🤔

Reply:
Ohh lol, didn't notice Vlad's comment. So sorry :+1: